include/libp2p/crypto/secp256k1_provider.hpp

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LIBP2P_CRYPTO_SECP256K1_PROVIDER_HPP
+#define LIBP2P_CRYPTO_SECP256K1_PROVIDER_HPP
+
+#include <array>
+
+#include <gsl/span>
+#include <libp2p/outcome/outcome.hpp>
+
+namespace libp2p::crypto::secp256k1 {
+
+  /**
+   * @brief Common types
+   */
+  using PrivateKey = std::array<uint8_t, 32>;
+  using PublicKey = std::array<uint8_t, 33>;
+  using Signature = std::vector<uint8_t>;
```

Comment:
does signature have arbitrary size of fixed one? If it is fixed please change type to array

Reply:
Signature length isn't fixed and is mutable between 70...72 bytes